### ISSUE TITLE
Should all lesson plans have links to solution pages? Should all links in the lesson be listed in materials?  If so, we need to do a careful read for that.

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#333]
I just updated `[this lesson plan](https://www.bootstrapworld.org/materials/fall2020/en-us/courses/algebra-wescheme/lessons/alg-simple-inequalities/index.shtml)` and noticed that [the introducing Sam page](https://www.bootstrapworld.org/materials/fall2020/en-us/courses/algebra-wescheme/lessons/alg-simple-inequalities/pages/sam-intro.html) is not listed in the materials at the top of the lesson and also has no link to the solution page.  The resources listed under materials and additional exercises all have links to the original and the solutions.

**Comment by schanzer** [comment]
Good call! All student-facing material should be linked from the lesson page. Solutions don't need to be (and perhaps _shouldn't_ be), since they are automatically linked in the teacher resources page.

**Comment by schanzer** [comment]
Renaming and re-assigning, now that solutions have their own issues.

**Comment by flannery-denny** [comment]
Confirming the task is:

1. Ignore whether or not solution-pages are listed in the materials sections?

2. Read all algebra lesson plans - make sure every workbook page / desmos activity / geogebra activity mentioned is listed in the materials page

3. Read all the DS lessons - delete the generalized workbook links and add links to all specific pages mentioned

@schanzer

**Comment by schanzer** [comment]
1. yep!
2. yep!
3. I'll do a big update before I go to bed tonight, so what's there is will be up to date. But truly, you shouldn't be looking there. You have the latest build! In general, you'll want to use that for things like this.
